.gitignore

```diff
@@ -16,5 +16,7 @@
 /target
 /.idea
 *.iml
+src/test/java/com/endava/soa_5/cucumber/hook
```

Comment:
Why added hook and runner in .gitignore ?

Reply:
done!